spec/searchworks_format_spec.rb

```diff
@@ -588,6 +588,17 @@
       @smods_rec.from_str(m)
       expect(@smods_rec.sw_genre).to eq ['Thesis/Dissertation']
     end
+    it "Government Document: typeOfResource 'text', genre 'government publication'" do
+      m = "<mods #{@ns_decl}><genre authority=\"marcgt\">government publication</genre><typeOfResource>text</typeOfResource></mods>"
+      @smods_rec.from_str(m)
+      expect(@smods_rec.sw_genre).to eq ['Government document']
+      m = "<mods #{@ns_decl}><genre authority=\"marcgt\">Government publication</genre><typeOfResource>text</typeOfResource></mods>"
+      @smods_rec.from_str(m)
+      expect(@smods_rec.sw_genre).to eq ['Government document']
+      m = "<mods #{@ns_decl}><genre authority=\"marcgt\">Government Publication</genre><typeOfResource>text</typeOfResource></mods>"
+      @smods_rec.from_str(m)
+      expect(@smods_rec.sw_genre).to eq ['Government document']
+    end
```

Comment:
since this will also work without the authority, should there be another test showing that (it assigning the genre when the genre element has no authority attrib?